### PR TITLE
Create users

### PR DESCRIPTION
**Pull request by hmunfru**
#### REVIEWERS

@flopezag @jesuspg 
#### DESCRIPTION
- It creates users and update quota for the different roles
#### TESTING

Test locally. To be tested by cloud-init

### CODE REVIEW

**Comment by jesuspg** [comment]
LGTM

File: fiwaretestbeddeploy/create_skuld_users.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# For those usages not covered by the Apache version 2.0 License please
+# contact with opensource@tid.es
+#
+import generate_users
+
+if __name__ == '__main__':
+    create = generate_users.GenerateUser("community_user", "community_user", "community_user", "community")
```
Comment (flopezag) [inline fiwaretestbeddeploy/create_skuld_users.py]: I prefer that you use the nomenclature:

user_name="community_user", password="community_user", tenant_name="community_user", role_name="community" 

when you call the method, it gives more light to this case when all the attributes have the same value...

File: requirements.txt [hunk 2]
```diff
@@ -12,6 +12,5 @@ python-neutronclient
 python-novaclient
 -e git+https://github.com/ging/python-keystoneclient.git#egg=python-keystoneclient
 
-requests!=2.9.0,>=2.8.1 # Apache-2.0
-
+requests
```
Comment (flopezag) [inline requirements.txt]: Are you sure that we need these packages? docopt I doubt, requests I am not sure...

File: scripts/keystone_work.sh [hunk 3]
```diff
@@ -35,7 +35,6 @@ sudo apt-get update
 sudo apt-get install -y pwgen
 
 pip install -r ~/fiware-testbed-deploy/requirements.txt --allow-all-external
-pip install -e git+https://github.com/ging/python-keystoneclient.git#egg=python-keystoneclient
```
Comment (flopezag) [inline scripts/keystone_work.sh:38]: Sorry, why not?

**Comment by flopezag** [comment]
Missing Unit Tests?

File: scripts/keystone_work.sh (hunk 3 above)
Comment (hmunfru) [inline scripts/keystone_work.sh:38]: it is in the requirements

**Comment by hmunfru** [comment]
It is not a component to have unit tests, since it only install software, and user openstack tools for configuring. And everything in cloud-init

File: scripts/keystone_work.sh (hunk 3 above)
Comment (flopezag) [inline scripts/keystone_work.sh:38]: Ah ok, NTC

**Comment by flopezag** [comment]
LGTM

**Pull request merged**